pegasus/sites.v3/hourofcode.com/public/css/front-page.css

```diff
@@ -114,3 +114,12 @@
 .microsoft {
   width: 160px;
 }
+
+.multiple-event-warning {
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.ast {
```

Comment:
I would rename this to something like `required-asterisk`. Just reading this css out of context it's unclear what ast is or why it's colored.